doc/install/mne_python.rst

```diff
@@ -87,36 +87,57 @@ conda to ``/home/user/anaconda3``)::
     Anaconda install tips (`Stack Overflow`_ results are often helpful)
     to fix these or other problems when ``conda`` does not work.
 
+.. _environment_file:
+
 Installing MNE-Python and its dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Once you have Anaconda installed, the easiest way to install
-MNE-Python is to use the provided `environment file`_ to install MNE-Python
-and its dependencies into a new conda environment::
+MNE-Python with all dependencies is update your base Anaconda environment:
 
-    $ curl --remote-name https://raw.githubusercontent.com/mne-tools/mne-python/master/environment.yml
-    $ conda env create -f environment.yml
+.. _base environment file: https://raw.githubusercontent.com/mne-tools/mne-python/master/environment_base.yml
 
-You can also use a web browser to download the required `environment file`_ if
-you do not have ``curl``. Then you should activate the environment::
+.. collapse:: |linux| Linux
 
-    $ conda activate mne
+   Use the `base environment file`_, e.g.::
 
-You can activate the environment (``conda activate mne``) each time you
-open a terminal, or add this activation command in your ``~/.bashrc`` or
-``~/.profile`` so that it happens automatically. You should now see that
-``python`` points to the one from your MNE environment::
+       $ curl --remote-name https://raw.githubusercontent.com/mne-tools/mne-python/master/environment_base.yml
+       $ conda env update --file environment_base.yml
 
-    $ which python
-    /home/user/anaconda3/envs/mne/bin/python
+.. collapse:: |apple| macOS
+
+   Use the `base environment file`_ and then update PyQt using :samp:`pip`, e.g.::
+
+       $ curl --remote-name https://raw.githubusercontent.com/mne-tools/mne-python/master/environment_base.yml
+       $ conda env update --file environment_base.yml
+       $ pip install "PyQt5>=5.10"
+
+.. collapse:: |windows| Windows
+
+   - Download the `base environment file`_
+   - Open an Anaconda command prompt
+   - :samp:`cd` to the directory where you downloaded the file
+   - Run :samp:`conda env update --file environment_base.yml`
+
+
+If you prefer an isolated Anaconda environment, instead of using
+:samp:`conda update` to modify your base environment above, you can instead create
+a new dedicated environment with :samp:`conda env create -n mne -f ...`.
```

Comment:
why the `...` in this command?  the `:samp:` directive allows ``:samp:`conda env create -n mne -f environment_{<platform>}.yml` `` and `<platform>` will get italicized (though in the latest commit it looks like you're not using platform-specific env files anymore, so maybe the point is moot?)